### PR TITLE
Update schema of UserProfile, UserProfileToShow, UserTournamentStats to have TrueSkill mean and stddev.

### PR DESCRIPTION
**Pull request by cookieben**

### CODE REVIEW

File: src/types.go [hunk 1]
```diff
@@ -34,22 +34,26 @@ type MatchWithKey struct {
...
-	Losses     int
-	JoinDate   time.Time
+	Tournament     string
+	Name           string
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
+	Wins           int
+	Losses         int
+	JoinDate       time.Time
```
Comment (chiang831) [inline src/types.go]: You can use a new struct to hold new data.
We don't want to break current table.

File: src/types.go [hunk 2]
```diff
@@ -34,22 +34,26 @@ type MatchWithKey struct {
...
-	Losses  int
-	Badges  []Badge
+	Name           string
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
+	FFAWins        int
+	Wins           int
+	Losses         int
+	Badges         []Badge
```
Comment (chiang831) [inline src/types.go]: You can use a new struct to hold new data.
We don't want to break current table.

File: src/types.go [hunk 3]
```diff
@@ -86,10 +90,12 @@ type Tournament struct {
...
-	Losses       int
-	Rating       float64
+	UserID         int64
+	TournamentID   int64
+	FFAWins        int
+	Wins           int
+	Losses         int
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
```
Comment (chiang831) [inline src/types.go]: I think you can deploy to a staging version and check whether the current data still works.

Comment (chiang831) [reply src/types.go]: https://cloud.google.com/appengine/docs/flexible/python/testing-and-deploying-your-app
gcloud app deploy --project PROJECT_ID --version VERSION_ID --no-promote

File: src/types.go (hunk 2 above)
Comment (cookieben) [reply src/types.go]: I made a test run and it seems work fine: https://8080-dot-5290404-dot-devshell.appspot.com/?authuser=1

And it's adding new fields on existing table, which should be ok.

File: src/types.go (hunk 3 above)
Comment (cookieben) [reply src/types.go]: Tested in https://cookieben-dev-dot-elo-site-201802.appspot.com/tournament/TTA.

**Pull request closed without merging**